Zeal-Operating-System/ZealOS
# Telnet

## Changes:

- [x] Telnet
  - [x] Connect to Telnet/BBS servers
    - [x] Access Zeal community BBS server on `bbs.zealos.net:4001` (has an IRC bridge)
  - [x] View ANSI art files and save them as DolDoc (alt+o, alt+s)
    - [x] Those saved DolDoc documents can then be used as [wallpapers](https://github.com/Zeal-Operating-System/ZealOS/pull/123) 
- [x] NetLog is toggleable by calling `NetLogToggle;` (I find it too distracting at times)
    - [x] The DesktopUI's [WallPaper.ZC](https://github.com/Zeal-Operating-System/ZealOS/pull/125/files#diff-c683c51d944468800f8c7ac2d5783c23a760da6d82143cf0066c86ec7b65101dR405-R409) also has a button to toggle it on/off 
## Verification:

1. Start from a clean slate (I did an entire new `git clone` of the repo elsewhere on my PC)
2. Checkout the `telnet` branch
3. Build ISO
4. Create new HDD (for Qemu in my case)
5. Verify
  - [x] ISO builds without any issue
  - [x] Installs without any issue (BIOS Bootloader version)
  - [x] Old features work as expected
  - [x] New features work as expected
